component/http/handler.go

```diff
@@ -174,3 +174,8 @@ func extractParams(r *http.Request) map[string]string {
 	}
 	return p
 }
+
+// ExtractParams extracts dynamic URL parameters using httprouter's functionality
+func ExtractParams(r *http.Request) map[string]string {
```

Comment:
Why not directly export the unexported one?

Reply:
I just thought it might buy us some leeway for if/when we change our router, or the Patron http request object.  But on the other hand, we'll cross that bridge when we get there.

Reply:
As far as I can see, the signatures are the same so there is no real value in this.

Reply:
You're right. I've removed that extra layering.